src/extension.ts

```diff
@@ -8,32 +8,25 @@ import { Bookmark, MAX_BOOKMARKS, NO_BOOKMARK_DEFINED } from "./Bookmark";
 import { Bookmarks } from "./Bookmarks";
 import { Sticky } from "./Sticky";
 
+const STATE_SVG_VERSION = "numberedBookmarksSvgVersion";
+const getFillColor = () => vscode.workspace.getConfiguration("numberedBookmarks").get("gutterIconFillColor", "#00ff25");
```

Comment:
Both functions needs to check for empty value `"'`, because if you clear the value in the new Settings UI, it will return empty, instead of the _default value_ defined in code. This happens because the new Settings UI adds an empty value, instead of removing the entry in `settings.json`

```
    "numberedBookmarks.gutterIconNumberColor": "",
    "numberedBookmarks.gutterIconFillColor": ""
```

Reply:
done